python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py

```diff
@@ -190,10 +190,10 @@ def get_table_objects() -> Dict[str, StaticTableDetails]:
 
     return {
         "feed_info": feed_info,
-        "trips": trips,
         "routes": routes,
         "stops": stops,
         "stop_times": stop_times,
+        "trips": trips,
```

Comment:
The new trigger for the `trips` table requires it to be loaded after the `stop_times` table.

Reply:
i think we should have a comment here to that effect.

Reply:
Added